roadmap/src/main/webapp/question.html

```diff
@@ -25,25 +33,69 @@
           </div>
         </div>
-        <form method="POST" class="row g-3 needs-validation" name="signupForm"  action="/question-data" onSubmit="return checkInput(this)" style="position:relative; top:55px;" novalidate>
+        <form method="POST" class="row g-3 needs-validation" name="signupForm" 
+         action="/question-data" onSubmit="return checkInput(this)" style="position:relative; top:55px;" novalidate>
           <div class="col-md-8 offset-2">
             <h3 style="margin-bottom:75px">Welcome, <strong>|*FNAME*|</strong></h3>
           </div>
-          <div class="form-outline btn-rounded form-floating col-md-8 offset-2">
-            <input type="text" class="form-control" name="message" placeholder="College" required>
+          <div class="form-group col-md-8 offset-2">
             <label class="form-label">What College Do You Attend?</label>
+            <input type="text" class="form-control" name="message" placeholder="College" 
+            required>
           </div>
-          <div class="form-outline btn-rounded form-floating col-md-8 offset-2">
-            <input type="text" class="form-control" name="major" placeholder="major" value="Computer Science" required disabled>
+          <div class="form-group col-md-8 offset-2">
             <label class="form-label">Major</label>
+            <input type="text" class="form-control" name="major" placeholder="major" 
+            value="Computer Science" required> 
           </div>
-          <div class="form-outline btn-rounded form-floating col-md-8 offset-2">
-            <input type="text" class="form-control" name="graduationDate" maxlength="4" placeholder="graduationDate" required>
+          <div class="form-group col-md-8 offset-2">
             <label class="form-label">Expected Graduation Date</label>
+            <input type="text" class="form-control" name="graduationDate" maxlength="4"
+             placeholder="graduationDate" required>
           </div>
-          <div class="col-8 offset-2 text-center">
-            <button type="submit" value="Submit" class="nav-link green-solid-btn btn-rounded" href="#">Confirm</button>
-          </div>
+          <div class="form-group offset-2 mb-5">
+            <label for="exampleFormControlInput1">Have you taken Data Structures?</label>
+            <div class="form-check ml-5">
+               <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+               <label class="form-check-label" for="flexCheckDefault">
+                <a href="roadmap.html">Yes</a>
+               </label>
+            </div>
+             <div class="form-check ml-5">
+               <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+               <label class="form-check-label" for="flexCheckDefault">
+                <a href="roadmap.html">Yes</a>
+               </label>
+            </div>
```

Comment:
Leave this section out.
We had the previous buttons that specified the numerous courses to complete this same task. So if you could return the green buttons that were on the /course-questions.html page I'll adjust them to complete the same task.

Reply:
I can remove that part, I hope you saw that I also took the other questions and added them at the bottom

Reply:
Yes I did thank you. For now just remove that section and I'll add the buttons on my end. Thanks so much